daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -124,72 +124,85 @@ func (l *JSONFileLogger) Log(msg *logger.Message) error {
 }
 
 func writeLog(l *JSONFileLogger) (int64, error) {
+
+	if err := checkCapacityAndRotate(l); err != nil {
+		return -1, err
+	}
+
+	i, err := l.buf.WriteTo(l.f)
+	if err != nil {
+		l.buf.Reset()
```

Comment:
hmm sorry am i misunderstood? Reset() will execute when only error is occur in `.WriteTo()`. so if Reset line is deleted and error has occur during `.WriteTo()` , buffer will growing.. as you mentioned in other PR.